### PR TITLE
feat: add cloud segment.io key

### PR DESCRIPTION
**Pull request by povilasv**
## Pull request description 

@exu added the secret.

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: build/api-server/Dockerfile [hunk 1]
```diff
@@ -23,6 +24,7 @@ RUN cd cmd/api-server; \
             -X github.com/kubeshop/testkube/pkg/telemetry.TestkubeMeasurementID=$ANALYTICS_TRACKING_ID \
             -X github.com/kubeshop/testkube/pkg/telemetry.TestkubeMeasurementSecret=$ANALYTICS_API_KEY \
             -X github.com/kubeshop/testkube/pkg/telemetry.SegmentioKey=$SEGMENTIO_KEY" \
+            -X github.com/kubeshop/testkube/pkg/telemetry.CloudSegmentioKey=$CLOUD_SEGMENTIO_KEY" \
```
Comment (exu) [inline build/api-server/Dockerfile:27]: quote is invalid!

**Review by exu — CHANGES_REQUESTED** [review]
quotes

File: pkg/telemetry/sender_sio.go [hunk 2]
```diff
@@ -8,9 +8,11 @@ import (
 )
 
 const SegmentioEnvVariableName = "TESTKUBE_SEGMENTIO_KEY"
+const CloudEnvVariableName = "TESTKUBE_CLOUD_API_KEY"
 
 // Brew builds can't be parametrized so we are embedding this one
 var SegmentioKey = "jELokNFNcLeQhxdpGF47PcxCtOLpwVuu"
+var CloudSegmentioKey = ""
```
Comment (vsukhin) [inline pkg/telemetry/sender_sio.go:15]: In HomeBrew it will remain empty

**Review by vsukhin — APPROVED** [review]
one note

**Review by exu — APPROVED** [review]

**Pull request merged**